### PR TITLE
fix(scanoss): Fix issue on form file field #7320

### PR DESCRIPTION
**Pull request by scanoss-qg**
Form field "file" was missing while sending wfp file

### CODE REVIEW

File: plugins/scanners/scanoss/src/main/kotlin/ScanOss.kt [hunk 1]
```diff
@@ -48,6 +48,7 @@ import org.ossreviewtoolkit.utils.common.VCS_DIRECTORIES
 
 // An arbitrary name to use for the multipart body being sent.
 private const val FAKE_WFP_FILE_NAME = "fake.wfp"
+private const val ARG_FIELD_NAME     = "file"
```
Comment (sschuberth) [inline plugins/scanners/scanoss/src/main/kotlin/ScanOss.kt:51]: Style nit: Please don't align the `=`, but just use one single space before (and after) `=`.

**Review by sschuberth — CHANGES_REQUESTED** [review]
Just some nits:
- Please amend your Git commit with a signoff (`git commit --amend -s`) to pass the DCO checks.
- Please use a [Conventional Commit](https://www.conventionalcommits.org/) type / scope; in this case I'd use `fix(scanoss): ...`.
- Please mention `Fixes #7320.` in the commit message body.

**Comment by sschuberth** [comment]
Thanks @scanoss-qg for the update. Please squash all your commits into one, and force-push to the source branch of this PR as we prefer atomic commits. This will also finally make the DCO check pass, as all (i.e. the only) commits in the PR are signed off.

**Comment by sschuberth** [comment]
Also, the commit message title and body need to be split across different lines, like

```
fix(scanoss): Fix issue on form file field

This fixes #7320.

Signed-off-by: scanoss-qg <quique.goni@scanoss.com>
```

**Comment by sschuberth** [comment]
@scanoss-qg something went wrong with squashing commits. The message now contains parts of other commit messages.

File: utils/spdx/src/main/resources/declared-license-mapping.yml [hunk 2]
```diff
@@ -444,6 +444,7 @@
 "The GNU General Public License (GPL), Version 2, With Classpath Exception": GPL-2.0-only WITH Classpath-exception-2.0
 "The GNU General Public License, Version 2": GPL-2.0-only
 "The GNU General Public License, v2 with FOSS exception": GPL-2.0-only WITH Universal-FOSS-exception-1.0
+"The GNU General Public License, v2 with Universal FOSS Exception, v1.0": GPL-2.0-only WITH Universal-FOSS-exception-1.0
```
Comment (sschuberth) [inline utils/spdx/src/main/resources/declared-license-mapping.yml:447]: This code should be be here.

**Comment by scanoss-qg** [comment]
Sorry Sebastian! Was my bad. I Will fix ASAP

**Comment by sschuberth** [comment]
> Sorry Sebastian! Was my bad. I Will fix ASAP

No worries 😉 Now the signoff is again missing in the commit, and the spaces are back 😁 Please just *amend* your existing commit and force-push again instead of adding a new commit.

**Review by sschuberth — APPROVED** [review]

**Comment by scanoss-qg** [comment]
Great News! Thanks for the guiding!

**Pull request merged**